regression-test/suites/load_p0/broker_load/test_broker_load_with_where.groovy

```diff
@@ -0,0 +1,179 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("test_broker_load_with_where", "load_p0") {
+    // define a sql table
+    def testTable = "tbl_test_broker_load_with_where"
+    
+    def create_test_table = {testTablex ->
+        def result1 = sql """
+            CREATE TABLE IF NOT EXISTS ${testTable} (
+                `k1` BIGINT NOT NULL,
+                `k2` DATE NULL,
+                `k3` INT(11) NOT NULL,
+                `k4` INT(11) NOT NULL,
+                `v5` BIGINT SUM NULL DEFAULT "0"
+            ) ENGINE=OLAP
+            AGGREGATE KEY(`k1`, `k2`, `k3`, `k4`)
+            COMMENT 'olap'
+            PARTITION BY RANGE(`k2`)
+            (PARTITION pbefroe202308 VALUES [('0000-01-01'), ('2023-09-01')),
+            PARTITION p202309 VALUES [('2023-09-01'), ('2023-10-01')),
+            PARTITION p202310 VALUES [('2023-10-01'), ('2023-11-01')),
+            PARTITION p202311 VALUES [('2023-11-01'), ('2023-12-01')),
+            PARTITION p202312 VALUES [('2023-12-01'), ('2024-01-01')),
+            PARTITION p202401 VALUES [('2024-01-01'), ('2024-02-01')))
+            DISTRIBUTED BY HASH(`k1`) BUCKETS 16
+            PROPERTIES (
+            "replication_allocation" = "tag.location.default: 2",
+            "is_being_synced" = "false",
```

Comment:
better not using dynamic partition properties in regression test, it may fail when time goes by.

Reply:
done